README.md

```diff
@@ -173,10 +173,33 @@ addr_test1vp2fg770ddmqxxduasjsas39l5wwvwa04nj8ud95fde7f7guscp6v
 </details>
 
 <details>
-  <summary>How to generate a delegated payment address from a stake key (<strong>payment-delegated.addr</strong>)</summary>
+  <summary>How to generate a payment address from a payment key hash (<strong>payment.addr</strong>)</summary>
 
 ```console
-$ cardano-address address delegation $(cat stake.xvk) < payment.addr > payment-delegated.addr
+$ cardano-address key hash < addr.xvk > addr.vkh
+addr_vkh12j28hnmtwcp3n08vy58vyf0arnnrhtavu3lrfdztw0j0jng3d6v
+$ cardano-address address payment --network-tag testnet < addr.vkh > payment.addr
+addr_test1vp2fg770ddmqxxduasjsas39l5wwvwa04nj8ud95fde7f7guscp6v
+```
+</details>
+
+
+<details>
+  <summary>How to generate a delegated payment address, ie. base address, from a stake key (<strong>base.addr</strong>)</summary>
```

Comment:
```suggestion
  <summary>How to generate a delegated payment address (<strong>base.addr</strong>)</summary> from a stake key and a payment address
```

I'm not sure — did we define the notion of "base address" somewhere? If not, then the phrase about "base address" is probably more confusing than helpful. 🤔

Reply:
yes, it is defined in shelley spec, reward account, enterprise address and base address and used for example in ledger CDDL - see https://github.com/input-output-hk/cardano-ledger/blob/master/eras/alonzo/test-suite/cddl-files/alonzo.cddl#L148-L151